cmd/builder/cli/cli.go

```diff
@@ -270,3 +279,16 @@ var builderCmd = &cobra.Command{
 		return nil
 	},
 }
+
+func writeAnalyzed(path string, logger *log.Logger) (files.Analyzed, error) {
+	analyzed := files.Analyzed{
+		RunImage: &files.RunImage{
+			TargetMetadata: &files.TargetMetadata{
+				OS:   "linux",
+				Arch: runtime.GOARCH,
```

Comment:
> GOARCH, GOOS, and GOROOT are recorded at compile time and made available by constants or functions in this package, but they do not influence the execution of the run-time system. 

I might be wrong but there's no other `cflinuxfs4` stack variant then the amd64. We can of course also read that from `/etc/os-release`.

Reply:
we're already doing pretty much the same in https://github.com/cloudfoundry/cnbapplifecycle/blob/2d0be2cc7a95005ce59354c8fc0b7e50931bd70a/pkg/staging/buildpacks.go#L28-L31

using `runtime.GOARCH` is equal to hardcoding to `amd64` as long as we only build for this architecture